app/assets/javascripts/components/search/material_header.js.coffee

```diff
@@ -29,7 +29,7 @@ window.SMaterialHeaderClass = React.createClass
         (span {className: 'superTiny'},
           (SGenericLink {link: material.links.edit})
         )
-      if material.links.external_edit_iframe?
+      if material.links.external_edit_iframe? and not material.lara_activity_or_sequence
```

Comment:
I understand this code, but I am wondering what the 'reason' for it is (probably I could read the PT story?).  

I am not groking the matrix of materials with `edit`, `external_edit_iframe`, and `lara_activity_or_sequence`